awx/ui/package-lock.json

```diff
@@ -258,6 +258,11 @@
         "rrule": "github:jkbrzt/rrule#4ff63b2f8524fd6d5ba6e80db770953b5cd08a0c"
       },
       "dependencies": {
+        "angular": {
```

Comment:
OK actually I was mistaken - I don't think this is necessary anymore.  angular-scheduler should now be requiring the same version of angular (1.7.2) that the rest of the app requires.  I'm fairly certain we should get rid of this.